suite-native/atoms/src/Select/Select.tsx

```diff
@@ -24,9 +24,10 @@ type SelectProps = {
     selectLabel: string;
     value: SelectValue;
     onSelectItem: (value: SelectValue) => void;
+    valueLabel?: string;
```

Comment:
I think it will be better to make this part of `SelectItemType`, because eveything else is in the Item but just this as separate prop?

Reply:
Yes but this is just displaying the label of the Select itself? Same as value kind of. I would then have to send the whole item to Select trigger or call `.find` on the items array and thought this is much cheaper to just have it as prop.